### PR TITLE
Input & image

### PR DESCRIPTION
**Pull request by jeetbhalani15**
image

### CODE REVIEW

File: components/cards/card.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
```
Comment (dev-enforced) [inline components/cards/card.html:1]: One more thing to clarify about: This branch is related to input & image right then the card related materials are also showing up so you can edit the name of the Pull request as cards,Input&image

File: components/cards/card.css [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    margin-top: 8px;
+}
+.overlay{
+    opacity: 50%
+
+}
+.text-overlay{
+    position: absolute;
+    top: 37%;
+    font-size: 24px;
```
Comment (dev-enforced) [inline components/cards/card.css:236]: Use of rems over px is preferred.so this is where you can make the appropriate changes.

File: components/avtar/avtar.css [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    border-radius: 10%;
+}
+
+/* 
+CSS FOR AVTAR WITH BADGES
+*/
+.container{
+    display: flex;
+    align-items: center;
+    gap: 30px;
```
Comment (dev-enforced) [inline components/avtar/avtar.css:71]: You can use rems here too.

File: components/cards/card.css [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+}
+i{
+    margin: 8px;
+}
+
+/* 
+vertical card
+*/
+.card-end{
+    display: flex;
```
Comment (dev-enforced) [inline components/cards/card.css:190]: You have used the property of display:flex;justify-content:center;align-items:center at multiple places so you can make an utility class of it and attach it to the respective element and then the rest of the properties you can add up.

**Review by dev-enforced — CHANGES_REQUESTED** [review]

**Review by Jay-Gadhiya — APPROVED** [review]

File: components/cards/card.css (hunk 4 above)
Comment (jeetbhalani15) [reply components/cards/card.css:190]: Ok done

File: components/avtar/avtar.css (hunk 3 above)
Comment (jeetbhalani15) [reply components/avtar/avtar.css:71]: Ok

**Review by gopalbbharadva — APPROVED** [review]
Try using variable names for color instead of hard coded hex value. 
use relative units.

**Pull request merged**